style.css

```diff
@@ -6,9 +6,13 @@ body {
     font-size: 30%;
   }
 }
-
-::-webkit-scrollbar {
-  display: none;
+.main::-webkit-scrollbar {
+  width:1.5rem;
+  padding:1rem;
+}
+.main::-webkit-scrollbar-thumb{
```

Comment:
You don't need to give browser-specific (`webkit`) styling as it is handled automatically.